azuredevops/internal/acceptancetests/testutils/hcl.go

```diff
@@ -766,3 +766,80 @@ resource "azuredevops_servicehook_storage_queue_pipelines" "test" {
 }
 `, projectResource, accountKey, queueName, eventType)
 }
+
```

Comment:
Can you move these configs from this file to its dedicated test file? This file is meant to only keep the configs that are likely shared by many other resources.